econtools/econ_workflow.py

```diff
@@ -0,0 +1,41 @@
+
+"""
+econ_workflow.py - starts arbitrary continuum publishing workflows
+"""
+
+from optparse import OptionParser
+import json
+import boto.sqs
+import boto.sqs.connection
+from econtools.aws import get_queue
+
+
+def start_workflow(queue_name, workflow_name):
+
+    queue = get_queue(queue_name)
+    message = {
+        'workflow_name': workflow_name,
+        'workflow_data': {}
```

Comment:
Specifically here, it would be good to see if ``workflow_data`` could be passed from the command line arguments somehow, but have a default of ``{}`` if not specified. I'm not sure how hard it would be to supply JSON as a command line argument to pass in here.

Reply:
It's not difficult to implement. I would however suggest moving to [argparse](https://docs.python.org/3/library/argparse.html) and moving away from [optparse](https://docs.python.org/3.5/library/optparse.html) as optparse is marked as deprecated since version 3.2. On the source page 

> The optparse module is deprecated and will not be developed further; development will continue with the argparse module.